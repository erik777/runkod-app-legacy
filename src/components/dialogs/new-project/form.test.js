import React from 'react';
import ProjectForm from './form';
import TestRenderer from 'react-test-renderer';


test('1- default', () => {
  const props = {
    switchFn: () => {
    },
    onDone: () => {
    }
  };

  const renderer = TestRenderer.create(
    <ProjectForm {...props}/>
  );

  expect(renderer.toJSON()).toMatchSnapshot();
});
