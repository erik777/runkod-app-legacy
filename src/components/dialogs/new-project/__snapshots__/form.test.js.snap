// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1- default 1`] = `
<div
  className="new-project-form"
>
  <div
    className="name-input"
  >
    <div
      className="input-group"
    >
      <input
        autoComplete="off"
        autoFocus={true}
        className="form-control"
        id="txt-name"
        maxLength={16}
        onChange={[Function]}
        placeholder="e.g., super-project"
        value=""
      />
      <div
        className="input-group-append"
      >
        <span
          className="input-group-text"
        >
          .runkodapps.com
        </span>
      </div>
    </div>
    <div
      className="form-feedback"
    >
      
    </div>
  </div>
  <div
    className="d-flex justify-content-between"
  >
    <button
      className="btn btn-link btn-sm"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      <u>
        Custom Domain
      </u>
    </button>
    <button
      className="btn btn-primary"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      Next
    </button>
  </div>
</div>
`;
